### DISCUSSION TITLE
Change mesh from one simulation to another

### DISCUSSION THREAD

**Discussion by krb-nantes** [idaholab/moose#22638] [Q&A Meshing] [answered]
Hi everybody!
I need to simulate a Zry cladding oxide spallation, making the oxide grow before spalling. So, I planned to first do the oxide growth on a whole block until the desired thickness and then, make a second simulation where I would remove part of this block (this would be the spalled part), but I wanted to use some data from the first simulation, as the temperature profile, by setting the "restar_file_base" on [Problem] block, however, I receive an error doing this, I must use the same mesh for both.
So, is there a way that I can make modifications in the mesh on the second simulation and use the data from the first one?

**Comment by GiudGiud** [comment, marked as answer]
Hello

You ll want to use the SolutionUserObject to do that.

Alternatively you can load the other mesh in a MultiApp and transfer information from there

Guillaume

**Reply by krb-nantes** [reply]
I was able to load the variable values and use as IC for the second simulation, however I could not retrieve the oxide thickness, because it is set as AuxVariable, and I cannot set IC for AuxVariable.

**Reply by GiudGiud** [reply]
use an auxkernel executed on INITIAL for an IC of an AuxVariable
